<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoodTracker</class>
 <widget class="QWidget" name="MoodTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mood Tracker</string>
  </property>
  <widget class="QLabel" name="MoodLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mood:</string>
   </property>
  </widget>
  <widget class="QSlider" name="MoodSlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="EnergyLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy:</string>
   </property>
  </widget>
  <widget class="QSlider" name="EnergySlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="AnxietyLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Anxiety:</string>
   </property>
  </widget>
  <widget class="QSlider" name="AnxietySlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JournalButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Journal </string>
   </property>
  </widget>
  <widget class="QPushButton" name="TasksButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Tasks </string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_1mood">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_mood">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MoodRecommendationLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Mood recommendation </string>
   </property>
  </widget>
  <widget class="QLabel" name="AnxietyRecommendationLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Anxiety recommendation </string>
   </property>
  </widget>
  <widget class="QLabel" name="EnergyRecommendationLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy recommendation </string>
   </property>
  </widget>
  <widget class="QLabel" name="PreviousMoodLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous mood:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PreviousEnergyLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous energy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PreviousAnxietyLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous anxiety: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
